### DISCUSSION TITLE
Color of Image using in IconOverride for back button is always turned into white.

### DISCUSSION THREAD

**Discussion by tongvietdung** [dotnet/maui#14182] [Q&A]
Hi everybody, I want my app has its own custom title bar with a custom logout button. This is what I do.
```
<ContentPage ...>
    <Shell.BackButtonBehavior>
        <BackButtonBehavior 
            Command="{Binding LogoutCommand}" IconOverride="{AppThemeBinding Dark=logout_icon_white.png, Light=logout_icon_white.png}"/>
    </Shell.BackButtonBehavior>

    <Shell.TitleView>
        <Grid
            Padding="9">
            <Grid.ColumnDefinitions>
                <ColumnDefinition Width="*"/>
                <ColumnDefinition Width="*"/>
            </Grid.ColumnDefinitions>

            <Image
                Source="app_logo.png"
                HorizontalOptions="End"
                Grid.Column="0"/>

            <Image
                Source="placeholder_icon.png" <!-- this is to make the logo above relatively in the center -->
                IsVisible="False"
                HorizontalOptions="End"
                Grid.Column="1"/>
        </Grid>
    </Shell.TitleView>
</ContentPage>
```

But I noticed that the color of the icon is always white no matter how the image files or the theme are. I've tried with different images with various colors but the back buton is always in white. Is there anything I'm missing?

**Comment by mrtcn** [comment]
@tongvietdung I see that no one suggested any solution till now, did you find any solution to that issue?

**Reply by tongvietdung** [reply]
Sorry I didn't have any solution to this. I have to create a dedicated button instead of using default back button.

**Reply by loncho67880** [reply]
I have the same problem, the Icon is there, but doesn't appear because the background is White and the Icon appear White although the png color is black.
![image](https://github.com/dotnet/maui/assets/21958802/d765145a-26ed-49e9-abb6-2f03e5924209)

**Reply by alessandrocaliaro** [reply]
@loncho67880 can you open an issue for this? I think conversations are not considered by the team to resolve bugs

**Reply by acaliaro** [reply]
https://github.com/dotnet/maui/issues/17228

**Reply by WillianPaboon** [reply]
Hello, have you find some solution?
